src/main/java/com/laytonsmith/core/events/drivers/ServerEvents.java

```diff
@@ -159,10 +163,7 @@ public String docs() {
 
 		@Override
 		public boolean matches(Map<String, Construct> prefilter, BindableEvent event) throws PrefilterNonMatchException {
-			if (event instanceof MCCommandTabCompleteEvent) {
-				return true;
-			}
-			return false;
+			return event instanceof MCCommandTabCompleteEvent;
```

Comment:
Hah. There's actually several of these. I fixed them when I was trying to improve prefilters, but I haven't had a reason to commit them. I usually just discard these types of changes, but this was related enough that I let it slide.